cts/cli/regression.crm_mon.exp

```diff
@@ -1286,7 +1286,261 @@ Negative Location Constraints:
+      <resource_history id="dummy" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="16" task="stop" rc="0" rc_text="OK" exec-time="6048ms" queue-time="0ms"/>
+      </resource_history>
+      <resource_history id="mysql-proxy" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="2" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+        <operation_history call="3" task="monitor" rc="0" rc_text="OK" interval="10000ms" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+      <resource_history id="promotable-rsc" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="2" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+        <operation_history call="4" task="monitor" rc="0" rc_text="OK" interval="10000ms" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+      <resource_history id="httpd-bundle-ip-192.168.122.131" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="2" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+        <operation_history call="3" task="monitor" rc="0" rc_text="OK" interval="60000ms" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+      <resource_history id="httpd-bundle-docker-0" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="2" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+        <operation_history call="3" task="monitor" rc="0" rc_text="OK" interval="60000ms" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+      <resource_history id="httpd-bundle-0" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="2" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+        <operation_history call="3" task="monitor" rc="0" rc_text="OK" interval="30000ms" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+    </node>
+    <node name="httpd-bundle-0">
+      <resource_history id="httpd" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="1" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+    </node>
+    <node name="httpd-bundle-1">
+      <resource_history id="httpd" orphan="false" removed="false" migration-threshold="1000000">
+        <operation_history call="1" task="start" rc="0" rc_text="OK" exec-time="0ms" queue-time="0ms"/>
+      </resource_history>
+    </node>
+  </node_history>
+  <bans>
+    <ban id="not-on-cluster1" resource="dummy" node="cluster01" weight="-1000000" promoted-only="false" master_only="false"/>
+  </bans>
+  <status code="0" message="OK"/>
+</pacemaker-result>
+Expecting element status, got summary
+Element pacemaker-result failed to validate content
+- fails to validate
```

Comment:
This looks like a problem.

Reply:
Glad you saw this. It got lost in the sea of red and green for me. Looking into it...

Reply:
I missed updating the `nodes` API schema to include the new `resources` schema.